### PR TITLE
Fix NPE due in convertJsonToObject(String)

### PR DESCRIPTION
**Pull request by coheneli1**

### CODE REVIEW

**Comment by coheneli1** [comment]
@velo the continuous-integration tests are failing due to "createdb nativejson" error (not related to this PR)

File: src/main/java/com/marvinformatics/hibernate/json/JsonUserType.java [hunk 1]
```diff
@@ -89,6 +89,9 @@ public Object nullSafeGet(ResultSet rs, String[] names, SessionImplementor sessi
     }
 
     Object convertJsonToObject(String content) {
+        if (content == null) {
+            return null;
```
Comment (velo) [inline src/main/java/com/marvinformatics/hibernate/json/JsonUserType.java]: Can you please include a unit test?

Comment (coheneli1) [inline src/main/java/com/marvinformatics/hibernate/json/JsonUserType.java]: @velo  sure, done.

**Comment by velo** [comment]
Ok, I gave up on appveyor... just get the build passing on travis and I will merge/release it

**Comment by velo** [comment]
Release in progress
https://github.com/velo/hibernate-native-json/issues/7

**Pull request merged**